### ISSUE TITLE
Custom template

### ISSUE THREAD

**Issue by Shuunen** [brrd/Abricotine#42]
Hi,

I looked at https://github.com/brrd/Abricotine/blob/master/docs/templates.md to create and use my own template.

I went to config directory with : 
![image](https://cloud.githubusercontent.com/assets/439158/13874311/5d63ceea-ecf7-11e5-81cd-a37a866aa73a.png)

Then in the template folder, I copy paste the default one and renamed it "custom" :
![image](https://cloud.githubusercontent.com/assets/439158/13874329/82007596-ecf7-11e5-8d16-a4614d59176a.png)

I edited template.json like this : 
```
{
    "name": "custom",
    "label": "Custom template",
    "accelerator": "CmdOrCtrl+E",
    "author": "Romain",
    "description": "Created from Abricotine Default Template",
    "license": "AGPL v3"
}
```

Then I edited the style.css with :
![image](https://cloud.githubusercontent.com/assets/439158/13874599/44caf88e-ecf9-11e5-9b98-fa48d8fd894a.png)

Then I edited the Abricotine config.json file to change the preview template :
![image](https://cloud.githubusercontent.com/assets/439158/13874554/05a9ac22-ecf9-11e5-8d8e-8e677eec64ef.png)

Then I restarted Abricotine and I still see the default render.

Did I missed a step ?

If I export using my custom template, it works : 
![image](https://cloud.githubusercontent.com/assets/439158/13874616/5fe5e494-ecf9-11e5-8d8f-dc7f57cc29ad.png)

![image](https://cloud.githubusercontent.com/assets/439158/13874624/69f0fab4-ecf9-11e5-96e6-87778acb5d2f.png)

**Comment by brrd** [comment]
> Then I restarted Abricotine and I still see the default render.

It seems that you are expecting a change in the editor. Actually `preview-template` key in `config.json` defines the template to use when you preview the doc in your browser (when you press F6), not to be confused with editor themes feature, which is not implemented yet (see #9).

**Comment by Shuunen** [comment]
Ah ok, so yes this is the feature I thought it was :)

Closing this because duplicate https://github.com/brrd/Abricotine/issues/9
